crates/iroha_cli/samples/parameter.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sumeragi": {
+    "block_time_ms": 2001
+  }
+}
```

Comment:
snake_case for consistency with `FindParameters` query result to cut and reuse it as `SetParameter` input

Reply:
If SDKs already implement `SetParameter` I'd revert this change @0x009922 @mversic

Reply:
Iroha JS supports this, but I am not sure I understand your point and what's the issue here. Could you help me understand?

Reply:
That is, for example,

```json
{
  "Sumeragi": {
    "BlockTimeMs": 2001
  }
}
```

will be failing to deserialize to `SumeragiParameter`

Reply:
Some `#[serde(...)]` attributes don't change the schema, so it seems easy for them to silently break the API

Reply:
if you want to break API, open a separate PR just for that. So that the change is in the focus